pkg/git/provider_ado_test.go

```diff
@@ -0,0 +1,132 @@
+package git
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	adoMock "github.com/argoproj-labs/argocd-autopilot/pkg/git/ado/mocks"
+	ado "github.com/microsoft/azure-devops-go-api/azuredevops/git"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"testing"
+)
+
+func Test_adoGit_CreateRepository(t *testing.T) {
```

Comment:
New to the mock libraries, any feedback or ways to this better are very welcomed